conf/cassandra.yaml

```diff
@@ -1889,6 +1889,9 @@ drop_compact_storage_enabled: false
 # Guardrail to allow/disallow DROP KEYSPACE statements
 # drop_keyspace_enabled: true
 #
+# Guardrail to allow/disallow bulk load of SSTables
+# bulk_load_enabled: true
```

Comment:
Can we add a note about the new guardrail in `NEWS.txt`? It seems it would be the very first new feature in 5.1 :)

It could look like:
```
5.1
===

New features
------------
    - New Guardrails added:
      - Whether bulk loading of sstables is allowed.
```

Reply:
OK so I guess the decision to put this to 5.1 was made? It would be great to have it in 5.0 but I can live with this.

This can be done on commit. @Runtian no worries about that.